sdk/digitaltwins/Azure.DigitalTwins.Core/tests/DigitalTwinClientTests.cs

```diff
@@ -15,50 +14,11 @@ public class DigitalTwinClientTests
         private const string PublicCloudScope = "https://digitaltwins.azure.net/.default";
```

Comment:
Consider renaming to "defaultCloudScope", since "public" is often used for a particular cloud.